### DISCUSSION TITLE
Can Seq be run as an web application in IIS?

### DISCUSSION THREAD

**Discussion by Sebazzz** [datalust/seq-tickets#1940] [Help]
Related to issue #810 regarding Let's Encrypt, there are great tools supporting Let's Encrypt in IIS now. I also noticed the Seq ships with a web.config with configuration for the ASP.NET Core module. That implies to me that it should be possible to have Seq running in IIS.

I guess it only comes down on configuring an application pool with the Seq service account user, and creating a website for Seq. Am I missing anything?

**Comment by nblumhardt** [comment]
Hi @Sebazzz, thanks for reaching out!

Seq doesn't include all of the infrastructure or configuration needed for running under IIS; I think it's very unlikely it will run successfully under IIS.

But, IIS can be configured as a reverse proxy; perhaps setting up your own IIS application with Lets Encrypt, and configuring it as a reverse proxy to Seq on the same box could get you close to what you're after?
